internal-packages/workflow-designer-ui/src/ui/card.tsx

```diff
@@ -0,0 +1,82 @@
+import clsx from "clsx";
+import * as React from "react";
+
+const Card = React.forwardRef<
```

Comment:
It looks like `forwardRef` is deprecated in React 19.  The recommended approach is to use the `ref` prop directly.

See the official React documentation for more details: [https://react.dev/reference/react/forwardRef](https://react.dev/reference/react/forwardRef)